libc/hdr/stdio_overlay.h

```diff
@@ -44,4 +55,14 @@
 #undef LIBC_SET_NO_INLINE
 #endif
 
+#ifdef LIBC_OLD_FORTIFY_LEVEL
+#define __USE_FORTIFY_LEVEL LIBC_OLD_USE_FORTIFY_LEVEL
```

Comment:
need `#undef __USE_FORTIFY_LEVEL` before re-defining it, because we defined it to be 0 on line 38.